clarify wording regarding subscription and unsubscription behavior (#29)
AMWA-TV/nmos-control-protocol: docs/Subscribing to events.md

# Subscribing to events

A controller can subscribe to all OIDs it is interested in receiving notifications from by using the [Subscription](https://specs.amwa.tv/is-12/branches/v1.0.x/docs/Protocol_messaging.html#subscription-message-type) message.

> The controller must always send the complete list of OIDs it wants to hold active subscriptions for. Omitting a previously subscribed OID from the request's `subscriptions` array signals that the controller wishes to unsubscribe from it.

Reviewer: `must` is a conformance language word here which would prevent us from adding this change to the current version.
I would suggest we change the wording:
```suggestion
Controllers always send the complete list of OIDs they want to hold active subscriptions for. Omitting a previously subscribed OID from the message's `subscriptions` array signals that the controller wishes to unsubscribe from it.
```

Author: sounds good!
